### PR TITLE
Pka extra tool

### PR DESCRIPTION
**Pull request by ba9f11ecc3497d9993b933fdc2bd61e5**
a shell script to use as an alternative to make-dns-cert.c (which I tried to use with a limited success)

the shell script does the same as make-dns-cert.c and returns you a resource record you can add to your dns zone file to indicate the location of your key (that you can retrieve with the option : --auto-key-locate pka)

### CODE REVIEW

File: tools/create-dns-pka.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+## Unsure why this offset is needed and present in make-dns-cert
+TOTAL_LENGTH=6
+
+function usage {
+        echo "Usage: "$0" -k [fingerprint of a key in your keyring] -d [dns record you want to create] -u [url where to fetch the key from]"
```
Comment (Shea690901) [inline tools/create-dns-pka.sh:7]: Shouldn't this read:
`        echo "Usage: "$0" -f [fingerprint of a key in your keyring] -d [dns record you want to create] -u [url where to fetch the key from]"`?
At least you're using `-f` both in line 11 and as argument for getopts in line 17

File: tools/create-dns-pka.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        echo "Example: create-dns-pka.sh -f 0BE0E990E02DF48CE0A1F199530843F3420AC0D3 -d bobdomain.com -u https://bobdomain.com/gpg/bobkey.pub"
+	echo "Will return : mzbi7kpidyqxg7gzxmpyf7b8n52fmm1n._pka   TYPE37  \# 67 0006 0000 00 14 0CE0E980E01DF48CE0B1F199540843F3420AD0D3 68747470733a2f2f3230303031332e6e65742f6770672f6c756340323030
+4031332e6e65742e707562 "
+        echo '   -f   fingerprint of a gpg key already imported in your local keyring.'
+	echo '   -d   the domain name where you want to publish this record'
```
Comment (Shea690901) [inline tools/create-dns-pka.sh:12]: What do you need this information for?
In line 23 you are assigning it to the variable `RR`, which is never used beside the `if` in line 34....

**Review by Shea690901 — CHANGES_REQUESTED** [review]

File: tools/create-dns-pka.sh (hunk 2 above)
Comment (ba9f11ecc3497d9993b933fdc2bd61e5) [reply tools/create-dns-pka.sh:12]: correct. I wrote this a while back and I don't recall why I added the -d option

File: tools/create-dns-pka.sh (hunk 1 above)
Comment (ba9f11ecc3497d9993b933fdc2bd61e5) [reply tools/create-dns-pka.sh]: correct

**Review by Shea690901 — APPROVED** [review]

File: tools/create-dns-pka.sh (hunk 1 above)
Comment (Shea690901) [reply tools/create-dns-pka.sh]: Looks much better now :)

**Pull request closed without merging**